src/api/subscription/content-types/subscription/schema.json

```diff
@@ -0,0 +1,30 @@
+{
+  "kind": "collectionType",
+  "collectionName": "subscriptions",
+  "info": {
+    "singularName": "subscription",
+    "pluralName": "subscriptions",
+    "displayName": "Subscription"
+  },
+  "options": {
+    "draftAndPublish": true
+  },
+  "pluginOptions": {},
+  "attributes": {
```

Comment:
Do we want to allow user to select which entity type to subscribe https://github.com/dev-launchers/strapiv4/blob/main/src/api/notification/content-types/notification/schema.json#L27?

Reply:
Yeah!